run-conf/.run/ingwer_server.run.xml

```diff
@@ -6,7 +6,7 @@
     <option name="SHMEM_ADDRESS" />
     <option name="HOST" value="localhost" />
     <option name="PORT" value="5005" />
-    <option name="AUTO_RESTART" value="true" />
+    <option name="AUTO_RESTART" value="false" />
     <RunnerSettings RunnerId="Debug">
       <option name="DEBUG_PORT" value="5005" />
       <option name="LOCAL" value="false" />
```

Comment:
This will be removed with the next version and has no further association with Ingwer